Mathlib/Analysis/SpecialFunctions/Log/PosLog.lean

```diff
@@ -91,25 +98,34 @@ theorem monotoneOn_posLog : MonotoneOn log⁺ (Set.Ici 0) := by
     simp only [this, and_true, ge_iff_le]
     linarith
 
+/-- The function `log⁺` commutes with taking powers. -/
+@[simp] lemma posLog_pow {n : ℕ} {x : ℝ} : log⁺ (x ^ n) = n * log⁺ x := by
+  by_cases hn : n = 0
+  · simp_all
+  by_cases hx : |x| ≤ 1
+  · simp_all [pow_le_one₀, (posLog_eq_zero_iff _).2]
+  rw [not_le] at hx
+  have : 1 ≤ |x ^ n| := by simp_all [one_le_pow₀, hx.le]
+  simp [posLog_eq_log this, posLog_eq_log hx.le]
+
 /-!
 ## Estimates for Products
 -/
+
 /-- Estimate for `log⁺` of a product. See `Real.posLog_prod` for a variant involving
 multiple factors. -/
-theorem posLog_mul {a b : ℝ} :
-    log⁺ (a * b) ≤ log⁺ a + log⁺ b := by
-  by_cases ha : a = 0
-  · simp [ha, posLog]
-  by_cases hb : b = 0
-  · simp [hb, posLog]
+theorem posLog_mul {x₁ x₂ : ℝ} : log⁺ (x₁ * x₂) ≤ log⁺ x₁ + log⁺ x₂ := by
```

Comment:
I'd argue this variable rename just makes things harder to read.

Reply:
Reverted variable names to `x` and `y` for readability. Thanks.